en/components/grid.md

```diff
@@ -24,27 +24,27 @@ The Grid is essentially a repeater of columns and rows showing data in a tabular
 > | Header | Contains all the cells in the header |
 > | Body | Contains all the cells in the body |
 
-After detaching you may add the number of headers you need to show all the dimensions of your data and as many records as you want to show in your design simply by duplicating the first row of data that you already have created.
+After detaching, you may add the number of headers you need to show all the dimensions of your data and as many records as you want to show in your design simply by duplicating the first row of data that you already have created.
 
 ### Cell Types
 
-The Grid provides three types of cells serving different data visualization purposes. The Header Cell is only one per column and appears at the top of the grid to display the textual description of the data in that particular column. The Body Cell is used to build the table displaying data records and may vary. The Summary Cell is used to create a section at the bottom of the Grid where column [Summaries](grid-summaries.md) are displayed for each dimension such as count, minimum and maximum value.
+The Grid provides three types of cells serving different data visualization purposes. The Header Cell is only one per column and appears at the top of the grid to display the textual description of the data in that particular column. The Body Cell is used to build the table displaying data records and may vary. The Summary Cell is used to create a section at the bottom of the Grid where column [Summaries](grid-summaries.md) are displayed for each dimension such as count, minimum, and maximum value.
 
 ![](../images/grid_cell_header.png)
 ![](../images/grid_cell_body.png)
 ![](../images/grid_cell_summary.png)
 
-### Items (Header Cell)
+#
```

Comment:
@StefanIvanov I doubt this was intentional?

Reply:
This was mistakenly removed and I have corrected it with the code gen pull request. Thanks for pointing out!